buildconfig/CMake/Packaging/mantidpython.in

```diff
@@ -30,9 +31,22 @@ if [ -n "${PYTHONPATH}" ]; then
     LOCAL_PYTHONPATH=${LOCAL_PYTHONPATH}:${PYTHONPATH}
 fi
 
-# Launch
-LD_PRELOAD=${LOCAL_PRELOAD} TCMALLOC_RELEASE_RATE=${TCM_RELEASE} \
-    LD_LIBRARY_PATH=${LOCAL_LDPATH} QT_API=pyqt \
-    PV_PLUGIN_PATH=${PV_PLUGIN_PATH} \
-    PYTHONPATH=${LOCAL_PYTHONPATH} \
-    @WRAPPER_PREFIX@${IPYTHON} $*@WRAPPER_POSTFIX@
+if [ -n "$1" ] && [ "$1" = "--classic" ]; then
+    shift
```

Comment:
Is it possible to avoid the duplication of the command here? It's hard to spot what is different about the call at a glance. Can we have a new variable something like `PROG` that is set based on the `if` and then there is a single call that uses `${PROG}` rather than `${PYTHON}` or `${IPYTHON}?`